modules/ossm-vs-istio.adoc

```diff
@@ -136,4 +88,4 @@ All OpenShift routes created by {ProductName} are in the `istio-system` namespac
 
 [id="ossm-cni_{context}"]
 == Container Network Interface (CNI)
-{ProductName} includes CNI which provides you with an alternate way to configure application pod networking. When you enable CNI, it replaces the `init-container` network configuration eliminating the need to grant service accounts and namespaces additional privileges by modifying their Security Context Constraints (SCCs). 
\ No newline at end of file
+{ProductName} includes CNI which provides you with an alternate way to configure application pod networking. CNI replaces the `init-container` network configuration eliminating the need to grant service accounts and projects additional privileges by modifying their Security Context Constraints (SCCs).
```

Comment:
"includes _a_ CNI _plugin_..."

"_The_ CNI _plugin_ replaces..."

Replace:
"eliminating the need to grant service accounts and projects additional privileges by modifying their Security Context Constraints (SCCs)."

with:

"eliminating the need to grant service accounts and projects access to Security Context Constraints (SCCs) with elevated privileges"

Reply:
Fixed.